test/torchaudio_unittest/common_utils/case_utils.py

```diff
@@ -75,3 +75,21 @@ def skipIfNoExtension(test_item):
     if 'TORCHAUDIO_TEST_FAIL_IF_NO_EXTENSION' in os.environ:
         raise RuntimeError('torchaudio C++ extension is not available.')
     return unittest.skip('torchaudio C++ extension is not available')(test_item)
+
+
+skipIfNoTransducer = unittest.skipIf(
+    not is_module_available('_warp_transducer'),
+    '"_warp_transducer" is not available',
+)
+
+try:
+    torch.ops.warprnnt_pytorch_warp_rnnt.gpu_rnnt
+    _CUDA_TRANSDUCER = True
```

Comment:
I'd make this switch dependent on whether CUDA [is available](https://pytorch.org/docs/master/cuda.html#torch.cuda.is_available) at all.